extensions/health/src/controllers/health.controller.ts

```diff
@@ -0,0 +1,50 @@
+// Copyright IBM Corp. 2019. All Rights Reserved.
+// Node module: @loopback/extension-health
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {HealthChecker} from '@cloudnative/health';
+import {bind, BindingScope, Constructor, inject} from '@loopback/core';
+import {get} from '@loopback/rest';
+import {HealthBindings} from '../keys';
+import {DEFAULT_HEALTH_OPTIONS, HealthOptions} from '../types';
+
+export function healthControllerFactory(
```

Comment:
Function names usually start with a verb.

```suggestion
export function createHealthController(
```